lumicks/pylake/kymotracker/detail/msd_estimation.py

```diff
@@ -1,5 +1,34 @@
 import numpy as np
 import warnings
+from dataclasses import dataclass
+
+
+@dataclass
+class DiffusionEstimate:
```

Comment:
something that just came to mind, but we might want to make this frozen from the get-go